glassfish-on-suse/README.md

```diff
@@ -0,0 +1,71 @@
+# GlassFish on SUSE
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https://raw.githubusercontent.com/CalCof/azure-quickstart-templates/master/glassfish-on-suse/azuredeploy.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a>
+<a href="http://armviz.io/#/?load=https://raw.githubusercontent.com/CalCof/azure-quickstart-templates/master/glassfish-on-suse/azuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+This template deploys GlassFish application server onto multiple load balanced SUSE Linux VMs. It is possible to select either OpenSUSE or SLES for the OS, and any release package associated with version 3 or 4 of GlassFish.
+
+This template will deploy the following resources:
+
+|Resource           |Properties                                                                                                                          |
+|-------------------|------------------------------------------------------------------------------------------------------------------------------------|
+|Virtual Network    |**Address prefix:** 10.0.0.0/16<br />**VM subnet:** 10.0.0.0/24                              |
+|Load Balancer      |Two probes and two rules for TCP 8080 and TCP 4848                                                                                       |
+|Public IP Addresses|Public IP attached to Load Balancer                                                |
+|Network Security Group|Two inbound rules                                                |
+|Storage Accounts   |One Storage Account                                                                                                                  |
+|Virtual Machines   |User defined number of VMs|
+
+### Generate SSH Keys
+
+You'll need to generate a pair of SSH keys in order to provision this template.<br/>
+If you are using a Windows computer, then you can download puttygen.exe to create a valid key pair. Alternativley, on a Linux or Mac, you can just use the ssh-keygen command.
+
+### azuredeploy.Parameters.json File Explained
```

Comment:
I have incorporated your comment, but as it was placed on the line above the parameters, it is not automatically hidden